modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/presubmit.yml

```diff
@@ -0,0 +1,20 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--cxxopt=-std=c++17'
```

Comment:
`/std:c++17` is the correct flag on Windows with MSVC. You could split this task into one for Windows and one for the rest of the matrix.